docker/edge/http/script.sh

```diff
@@ -98,13 +152,17 @@ if [ $? -eq 0 ]; then
      continue
    fi
 done
-#cat access_token.txt 
+
 access=`cat access_token.txt |jq . | grep "access_token" | cut -d ":" -f2 | sed -e 's/ //g' -e 's/"//g' -e 's/,//g' -e 's/\\n//g' | tr -d '\n'`
 echo -----------------------------
 echo " access token is" $access 
 echo -----------------------------
 
-echo "end of script"
-rm -rf accessAPI.txt access_token.txt generate_token.txt
-
+#To generate Authorization Token
 
+AUTH=`echo -n $ID:$SECRET | base64 | tr -d "\t\r\n"` 
+echo ------------------------------------------
+echo "Authorization : Basic "$AUTH 
+echo ------------------------------------------
+echo "end of script"
```

Comment:
[minor] Better to explain about which script ended. like authentication script.

Reply:
Incorporated proper comment